cms/startup.py

```diff
@@ -50,6 +52,9 @@ def run():
     xmodule.x_module.descriptor_global_handler_url = cms.lib.xblock.runtime.handler_url
     xmodule.x_module.descriptor_global_local_resource_url = xblock_local_resource_url
 
+    if settings.FEATURES.get('ENABLE_MKTG_SITE', False):
```

Comment:
@wajeeha-khalid why do we need to pass **False** as a default value here?
if ``ENABLE_MKTG_SITE`` variable does not exist, this check will return None by default.